Libplanet/Action/TotalSupplyNotTrackableException.cs

```diff
@@ -41,6 +41,15 @@ private TotalSupplyNotTrackableException(SerializationInfo info, StreamingContex
         /// </summary>
         public Currency Currency { get; }
 
+        public static TotalSupplyNotTrackableException WithDefaultMessage(Currency currency)
```

Comment:
IMHO it doesn't have to be `public`.  How about making it `internal`?

Reply:
Amended!